api-inference-community/docker_images/keras/requirements.txt

```diff
@@ -1,5 +1,8 @@
 starlette==0.14.2
 api-inference-community==0.0.8
 numpy>=1.17
-tensorflow>=2.3
-huggingface_hub===0.0.13
\ No newline at end of file
+tensorflow==2.6.0
+# huggingface_hub===0.0.13
```

Comment:
@LysandreJik  Can we perhaps do dev release before we merge this so we don't have to have git install here?

Reply:
Yes! Will release in a minute.